concepts/05 UI Components/Drawer/Getting Started with Navigation Drawer/Getting Started with Navigation Drawer.md

```diff
@@ -1,14 +1,14 @@
 #include tutorials-intro-installationnote
 
-The Drawer is a dismissible or permanently visible panel used for navigation in responsive web application layouts.
+The Drawer is a dismissible or permanently visible panel that you can use for navigation in responsive web application layouts.
 
 DevExtreme supplies application templates for <a href="https://devexpress.github.io/devextreme-angular-template/#/home" target="_blank">Angular</a>, <a href="https://devexpress.github.io/devextreme-vue-template/#/home" target="_blank">Vue</a>, and <a href="https://devexpress.github.io/devextreme-react-template/#/home" target="_blank">React</a>. They implement a responsive layout that uses the Drawer. You can use these templates instead of following the tutorial. Refer to the following articles for more information:
```

Comment:
```suggestion
DevExtreme supplies application templates for <a href="https://devexpress.github.io/devextreme-angular-template/#/home" target="_blank">Angular</a>, <a href="https://devexpress.github.io/devextreme-vue-template/#/home" target="_blank">Vue</a>, and <a href="https://devexpress.github.io/devextreme-react-template/#/home" target="_blank">React</a>. They implement a responsive layout that uses the Drawer. You can use these templates instead of following the tutorial. Refer to the articles below for more information:
```